.devcontainer/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/devcontainers/miniconda:0-3
+
+# Copy environment.yml (if found) to a temp location so we update the environment. Also
+# copy "noop.txt" so the COPY instruction does not fail if no environment.yml exists.
+# COPY environment.yml* .devcontainer/noop.txt /tmp/conda-tmp/
+# RUN if [ -f "/tmp/conda-tmp/environment.yml" ]; then umask 0002 && /opt/conda/bin/conda env update -n base -f /tmp/conda-tmp/environment.yml; fi \
+#     && rm -rf /tmp/conda-tmp
+
+COPY tests/requirements.txt /tmp/conda-tmp/
```

Comment:
If the intention is for this container to be used for testing (e.g., to run pytest) we will need to also copy and install from `tests/requirements-Linux.txt` and `tests/requirements-ci.txt`

see `dev/start`:

https://github.com/conda/conda/blob/0f8fb581cdfb1716b22596dace01747bc978eeb1/dev/start#L241-L253